z3-sys/build.rs

```diff
@@ -17,6 +19,33 @@ fn main() {
     };
     println!("cargo:rerun-if-env-changed={}", Z3_HEADER_VAR);
     println!("cargo:rerun-if-changed={}", header);
+
+    generate_binding(&header);
+}
+
+#[cfg(feature = "vcpkg")]
```

Comment:
I'd rather rename `build_z3` to `build_bundled_z3` and then add a new function that uses `vcpkg` and just call it from the existing `main` ...

Reply:
This would also take care of things like the missing `rerun-if-changed` stuff in the new vcpkg-specific copy of `main`.

Reply:
Yes, I've renamed it and added a `rerun-if-changed`. And related env var should have been watched inside vcpkg-rs. Thanks.